docs/source/tutorials/basic_renewal_model.qmd

```diff
@@ -252,7 +252,38 @@ Now, let's investigate the output, particularly the posterior distribution of th
 ```{python}
 # | label: fig-output-rt
 # | fig-cap: Rt posterior distribution
-out = model1.plot_posterior(var="Rt")
+import arviz as az
+
+idata = az.from_numpyro(
+    posterior=model1.mcmc,
+)
+
+# Extract Rt signal
+rt = idata.posterior["Rt"]
+rt = rt.mean(dim="chain")
+rt = rt.T
```

Comment:
I don't understand why you take the mean here?

Reply:
I could have used `rt.squeeze()` if there was a single chain but there were 2 chains.

Reply:
but why do you want to take the mean? The legend makes it look like you want to plot individual samples.

Reply:
Okay, then would you like me to choose chain 0 or 1? Or, would you like me to update the legend?

Reply:
In general for this sort of spaghetti plot you:
- want to plot individual samples as individual pieces of spaghetti
- in that context, want to treat chain 0 samples, chain 1 samples, etc equivalently, and plot (typically a random subset of) samples from all chains.

There are cases in which you might want to plot samples from different chains in distinguishable ways (e.g. for non-convergence debugging), but those are beyond the scope of this example.

Reply:
Agree with Dylan. You can randomly sample from all samples with `az.extract(idata, num_samples=100)`.

Reply:
Okay, I like this solution, will do.

Reply:
> There are cases in which you might want to plot samples from different chains in distinguishable ways (e.g. for non-convergence debugging), but those are beyond the scope of this example.

Yes. You may sometimes want to
1. compute a summary (e.g. mean) across all samples in each chain to verify the different chains produce similar results
2. compute a summary across all samples and all chains to see the overall posterior

Computing a summary across all chains only (not the samples within a chain) doesn't sound immediately useful to me.

Reply:
The mean is as though you concatenated vertically the samples in both chains. But yes, I see that extracting N samples from across the chains and plotting them individually is what is desired.